packages/process/_process.pony

```diff
@@ -0,0 +1,145 @@
+use @pony_os_errno[I32]()
+
+primitive _STDINFILENO
+  fun apply(): U32 => 0
+
+primitive _STDOUTFILENO
+  fun apply(): U32 => 1
+
+primitive _STDERRFILENO
+  fun apply(): U32 => 2
+
+primitive _SIGKILL
+  fun apply(): I32 => 9
+
+primitive _SIGTERM
+  fun apply(): I32 => 15
+
+// Operation not permitted
+primitive _EPERM
+  fun apply(): I32 => 1
+
+// No such process
+primitive _ESRCH
+  fun apply(): I32 => 3
+
+// Interrupted function
+primitive _EINTR
+  fun apply(): I32 => 4
+
+// Try again
+primitive _EAGAIN
+  fun apply(): I32 =>
+    ifdef bsd or osx then 35
+    elseif linux then 11
+    elseif windows then 22
+    else compile_error "no EAGAIN" end
+
+// Invalid argument
+primitive _EINVAL
+  fun apply(): I32 => 22
+
+interface _Process
+  fun kill()
+  fun ref wait(): I32
+
+class _ProcessNone is _Process
+  fun kill() => None
+  fun ref wait(): I32 => 0
+
+class _ProcessPosix is _Process
+  let pid: I32
+
+  new create(path: String, args: Array[String] val, vars: Array[String] val, stdin: _Pipe, stdout: _Pipe, stderr: _Pipe) ? =>
```

Comment:
Small formatting nitpick: can you make this a multiline method signature, following the style shown [here](https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#method-declarations)?

Reply:
yup